mapper/mapper_test.go

```diff
@@ -19,4 +19,20 @@ func TestUnitMapper(t *testing.T) {
 		So(geoStartModel.Page.Breadcrumb[0].Title, ShouldEqual, "Home")
 		So(geoStartModel.Page.Breadcrumb[0].URI, ShouldEqual, "/")
 	})
+
+	Convey("test CreateAreaPage mapper correctly initialises an AreaModel", t, func() {
+		areaModel := CreateAreaPage(mdl)
+
+		So(areaModel.BetaBannerEnabled, ShouldBeTrue)
+		So(areaModel.Metadata.Title, ShouldEqual, areaModel.Name + " Summary")
+		So(len(areaModel.Page.Breadcrumb), ShouldEqual, len(areaModel.Ancestors) + 2)
```

Comment:
You can measure length of an array by using `ShouldHaveLength`. This line and line 18 can be refactored to use it
```suggestion
				So(areaModel.Page.Breadcrumb, ShouldHaveLength, 2)
```

Reply:
I didn't know that assertion existed, thanks for the pointer